src/samples/press-package/v4/complete.json

```diff
@@ -36,6 +55,52 @@
             "statusDate": "2016-10-11T00:00:00Z",
             "volume": 5,
             "elocationId": "e18726"
+        },
+        {
+            "id": "19560",
+            "type": "reviewed-preprint",
+            "doi": "10.7554/eLife.19560",
+            "status": "reviewed",
+            "authorLine": "Lee R Berger et al.",
+            "title": "<i>Homo naledi</i>, a new species of the genus <i>Homo</i> from the Dinaledi Chamber, South Africa",
+            "indexContent": "<i>Homo naledi</i>, a new species of the genus <i>Homo</i> from the Dinaledi Chamber, South Africa",
+            "titlePrefix": "Title prefix",
+            "stage": "published",
+            "published": "2022-08-01T00:00:00Z",
+            "reviewedDate": "2022-08-01T00:00:00Z",
```

Comment:
If this is a `complete.json` file, would it be good to include a `versionDate` here too?

Reply:
> If this is a `complete.json` file, would it be good to include a `versionDate` here too?

Despite the `complete.json` label it's rarely complete where related content is concerned. I, at least, try to ensure one of every top level property declared in the model is available. I normally, when introducing a property, just start from the previous version samples and add the new thing. I've added `versionDate` anyway here.